src/com/company/reverse/ReverseTest.java

```diff
@@ -0,0 +1,31 @@
+package com.company.reverse;
```

Comment:
please move tests to specific directory for tests

Reply:
https://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html